### DISCUSSION TITLE
Refund Request : Lifetime License $18

### DISCUSSION THREAD

**Discussion by djnoxx-ai** [aclap-dev/video-downloadhelper#1112] [License & Purchase requests]
### What's your issue?

I recently purchased a license for DownloadHelper, expecting it to facilitate downloading videos from various sites. Unfortunately, I have discovered that it does not support this function for the sites I use.

I need a solution that can download these videos. Therefore, I kindly request a refund for the DownloadHelper license.

Thank you for your understanding. Please advise on the refund process.

E-Mail : harshkherwal5555@gmail.com

### The 10 first digits (only 10) of your license number or email address

harshkherwal5555@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
